ci/hardware_tests/device/device.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env -S python3 -u
```

Comment:
Let's remove this shebang. It seems to me this file is only `import`ed and never executed as a standalone binary.

Plus it's cleaner for a caller to set [`PYTHONUNBUFFERED`](https://docs.python.org/3/using/cmdline.html#envvar-PYTHONUNBUFFERED) env variable if they want to achieve this.

Reply:
Done.